docs/generate.js

```diff
@@ -12,6 +12,18 @@ try {fs.mkdirSync("./dist/archive/v" + version)} catch (e) {/* ignore */}
 var guides = fs.readFileSync("docs/nav-guides.md", "utf-8")
 var methods = fs.readFileSync("docs/nav-methods.md", "utf-8")
 
+const encodeHTML = function(str) {
+    const charsToEncode = /[&"'<>]/g
+    const encodeTo = {
+        "&": '&amp;',
+        "\"": '&quot;',
+        "'": '&#39;',
+        "<": '&lt;',
+        ">": '&gt;',
+    }
+    return str.replace(charsToEncode, function(char) { return encodeTo[char] })
+}
+
```

Comment:
This list of chars is chosen in response to the list of XML encodings in [wikipedia](https://en.wikipedia.org/wiki/Character_encodings_in_HTML), of which @pygy's list looks to be a close subset, but if any other characters should be added, let me know.